app/Traits/Statable.php

```diff
@@ -36,6 +36,16 @@ public function getStateMachine()
         return $this->stateMachine;
     }
 
+    /**
+     * gets the prior state associated with the Stateable model.
+     *
+     * @return self::HISTORY_MODEL
+     */
+    public function getPriorState()
+    {
+        return $this->history()->get(null)->last();
```

Comment:
Cool is the output predictable?

Reply:
yes, unlike `latest()` which can't tell between vouchers `created_at` in the same second, fetching the history orders by ID for `last()` to pop the end off. It shouldn't be a memory problem, in theory, as there shouldn't be very many transitions.